src/lib/api.js

```diff
@@ -1,8 +1,22 @@
 export function fetchCharacters() {
-  const urlCharacters = "https://rickandmortyapi.com/api/character";
-  const main = document.querySelector("main");
+  let url;
+  const filterDropdown = document.querySelector(".header__filter");
+  const type = filterDropdown.value;
+  const searchBar = document.querySelector(".header__search-bar");
+  const name = searchBar.value;
+  console.log(name);
+  if (type === "alive") {
```

Comment:
You can put this into its own function, that returns the correct url depending on the status of the characters. Then call it here